dds/DCPS/ValueWriter.h

```diff
@@ -9,6 +9,7 @@
 #include "Definitions.h"
 
 #include <dds/Versioned_Namespace.h>
+#include "XTypes/MemberDescriptor.h"
 #include <FACE/Fixed.h>
```

Comment:
```suggestion
#include "Definitions.h"
#include "XTypes/MemberDescriptor.h"

#include <dds/Versioned_Namespace.h>
#include <FACE/Fixed.h>
```

Reply:
Aren't `Versioned_Namespace.h`, `XTypes/MemberDescriptor.h`, and `FACE/Fixed.h` all in the "Headers from external OpenDDS-based libraries."?

Reply:
That's probably worded poorly, but by "external OpenDDS-based libraries" I meant things like Bench and other non-core libraries that use the core libraries. All of these headers are part of the core libraries.